terraform/aws/main.tf

```diff
@@ -201,8 +218,8 @@ locals {
 }
 
 resource "random_shuffle" "dns_ips" {
-  input        = concat(aws_instance.masternode.*.public_ip)
-  result_count = length(concat(aws_instance.masternode.*.public_ip)) > local.dns_record_length ? local.dns_record_length : length(concat(aws_instance.masternode.*.public_ip))
+  input        = concat(aws_instance.masternode_amd.*.public_ip)
+  result_count = length(concat(aws_instance.masternode_amd.*.public_ip)) > local.dns_record_length ? local.dns_record_length : length(concat(aws_instance.masternode_amd.*.public_ip))
```

Comment:
Does this mean only IPs from AMD hosts can appear in the DNS list?

Reply:
Fixed